### PR TITLE
Installing istio for perf testing

### PR DESCRIPTION
**Pull request by utka**
Use tools/perf/istio-install install istio for perf testing.

### CODE REVIEW

File: prow/setup_perf_cluster_for_tests.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+# Check https://github.com/istio/test-infra/blob/master/boskos/configs.yaml
+# for existing resources types
+export RESOURCE_TYPE="${RESOURCE_TYPE:-gke-perf-preset}"
+export OWNER="${OWNER:-perf-tests}"
+export PILOT_CLUSTER="${PILOT_CLUSTER:-}"
+export USE_MASON_RESOURCE="${USE_MASON_RESOURCE:-True}"
+export CLEAN_CLUSTERS="${CLEAN_CLUSTERS:-True}"
+# This is config for postsubmit cluster under istio/tools/perf/istio.
+export VALUES="${VALUES:-values-istio-postsubmit.yaml}"
```
Comment (howardjohn) [inline prow/setup_perf_cluster_for_tests.sh]: Doesn't this need to be an absolute path?

Comment (utka) [reply prow/setup_perf_cluster_for_tests.sh]: No, this is to override the values in this file: https://github.com/istio/tools/blob/4971a29534cae29ccf45486750524d8bbe26e75e/perf/istio-install/setup_istio.sh#L108

Comment (howardjohn) [reply prow/setup_perf_cluster_for_tests.sh]: Oh didn't realize we change directories to the install directory

**Pull request merged**